public/app/features/alerting/unified/components/contact-points/components/Modals.tsx

```diff
@@ -34,10 +38,14 @@ export const useDeleteContactPointModal = (
   const handleSubmit = useCallback(() => {
     if (contactPoint) {
       handleDelete(contactPoint)
-        .then(() => setShowModal(false))
+        .then(() => {
+          setShowModal(false);
+          // Notify other tabs subscribed to alerting broadcast channel that alert manager has been updated
+          postMessage(AlertingTabMessageTypes.AlertManagerUpdated);
```

Comment:
This could lead to unexpected changes in the alert rule form when user does not want to change anything.Any time user creates a contact point, then the selector would be updated...